src/config.c

```diff
@@ -344,7 +459,24 @@ CONFIG_SETTER(setFrisoINI) {
   RETURN_STATUS(acrc);
 }
 CONFIG_GETTER(getFrisoINI) {
-  return config->frisoIni ? sdsnew(config->frisoIni) : NULL;
+  if (config->frisoIni && strlen(config->frisoIni) > 0) {
```

Comment:
Doesn't this change the behaviour?
We no longer return empty strings?
Are we sure that is what we want?

If this is supposed to be backported then maybe we shouldn't break existing behaviour in that regard.